Gemfile

```diff
@@ -4,6 +4,8 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 gemspec
 
+gem 'sequel', github: 'jeremyevans/sequel'
```

Comment:
I wonder if we should set up a matrix with sequel from master + latest released version?